docs/getting-started.md

```diff
@@ -216,75 +216,58 @@ Troubleshooting tips if your EV3 won't boot
 
 ## Step 5: Set up a network connection
 
-There are lots of choices here. Your choice depends on what type of connection
-you want to use and on the OS of your host computer, so pick the one that applies:
-
-* __Ethernet over USB__ (just requires the USB cable that comes with the EV3)
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-usb) tutorial
-* __USB Ethernet dongle__ (as in the kind with an RJ45 connector)
-    * If your network has a DHCP server, this will "just work".
-* __USB Wi-Fi Dongle__
-    * Setting up Wi-Fi Via the Brickman User Interface (hopefully you can figure
-      this out, but it would be nice if someone made a tutorial)
-    * [Setting Up Wi-Fi Using the Command Line](../tutorials/setting-up-wifi-using-the-command-line)
-      (requires another connection type first to get to the command line, but it's way
-      easier to enter your passphrase this way)
-* __Bluetooth__
-    * Note: Bluetooth may not work on the first boot. Please reboot if you see "???"
-      after you power on Bluetooth.
-    * [Connecting to the Internet](../tutorials/connecting-to-the-internet-via-bluetooth) tutorial
+There are lots of choices here. Choose the connection option that best suits you
```

Comment:
It might be good to say "click on the button below" here since the "lots of choices here" are not actually *here* on this page, they are on another page.